package.json

```diff
@@ -34,7 +34,6 @@
     "lint": "eslint --ext .js --ext .jsx src/"
   },
   "devDependencies": {
-    "eslint": "^5.3.0",
```

Comment:
Did the update force this out? or a reason why we're removing it? I know CRA has linting by default not sure if eslint is a hidden thing like web pack, and this was a redundant add to begin.